### ISSUE TITLE
 Microsoft.Web/sites 2020-12-01 apiVersion warning

### ISSUE THREAD

**Issue by Elyseum** [Azure/azure-resource-manager-schemas#1781]
Latest version is 2020-12-01:
https://docs.microsoft.com/en-us/azure/templates/microsoft.web/2020-12-01/sites?tabs=json

But for "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#", the last version available is '2018-11-01'.

VS / VSCode gives following warning:
"consider updating the value to one of the following: "2016-08-01", "2014-06-01", "2015-08-01", "2018-02-01", "2018-11-01". It is possible that the current resource version is valid but that a schema has not been generated."

**Comment by apclouds** [comment]
@anthony-c-martin do you know if this error is coming from our side?

**Comment by anthony-c-martin** [comment]
> @anthony-c-martin do you know if this error is coming from our side?

Yes - this is the message you get when the VSCode extension doesn't know about the schema. This schema was published a while back (https://github.com/Azure/azure-resource-manager-schemas/commits/main/schemas/2020-10-01/Microsoft.Web.json). @StephenWeatherford may be able to confirm if it's been bundled in the latest ARM Tools release or not. If not, then it might be worth pushing an update to pick it up.

**Comment by StephenWeatherford** [comment]
The last snapshot was June 25, so we missed this.  We're currently blocked for picking up new schemas by https://github.com/microsoft/vscode-azurearmtools/issues/1324.  As soon as that is fixed, we'll try picking it up, hopefully within 2-3 weeks.
